.github/workflows/ci.yml

```diff
@@ -3,8 +3,12 @@ name: CI
 on:
   push:
     branches: [ main ]
+    paths-ignore:
+      - '**/*.md'
```

Comment:
I think we should leave it on for `main`, since it doesn't really speed things up.

Reply:
ok